cli/ballerina-tool/src/main/java/org/ballerinalang/tool/Main.java

```diff
@@ -54,7 +53,6 @@
 
     public static void main(String... args) {
         try {
-            ToolUtil.checkForUpdate(outStream, args);
```

Comment:
Can we create an issue to track this removal and where to put it instead? I mean, which command(s) to put this check for upcoming release ?

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/20021